aztk/config.py

```diff
@@ -31,6 +31,27 @@ def load_spark_config():
     except Exception as e:
         pass
 
+    try:
+        copyfile(
```

Comment:
This is fine for now, but once the SDK is used, this will be changed. None of these config files / jars should be written to the clients disk again. They should just be zipped up with node_scripts/.